content/v2.3/actions/formats-and-mime-types.md

```diff
@@ -7,35 +7,36 @@ Hanami maps [over 50 of the most common MIME types][built-in-formats] to simple
 
 [built-in-formats]: https://github.com/hanami/controller/blob/dc5bb2a1db48b0ccf3faf52aac20eaef0fd135a3/lib/hanami/action/mime.rb#L15-L69
 
-Configuring one or more formats for your actions will:
+Configuring your actions to accept one or more formats will:
 
-- Ensure the actions accept only appropriate requests based on their `Accept` or `Content-Type` headers
-- Set an appropriate `Content-Type` header on responses
+- Ensure the actions accept only matching requests based on their `Accept` or `Content-Type` headers
+- Default to an appropriate `Content-Type` header on responses
 - For certain formats, enable automatic parsing of request bodies
 
-## Configuring a format for all actions
+## Accepting a format across all actions
 
-To configure a format for all actions, use `config.actions.format` in your app class.
+To accept a format for all actions, use `config.actions.format` in your app class.
 
 ```ruby
 # config/app.rb
 
 module Bookshelf
   class App < Hanami::App
-    config.actions.format :json
+    config.actions.formats.accept :json
   end
 end
 ```
 
-You can also configure actions to use multiple formats:
+You can also configure actions to accept multiple formats:
 
 ```ruby
-config.actions.format :json, :html
+config.actions.formats.accept :json, :html
 ```
 
-## Configuring a format for particular actions
+## Accepting a format for particular actions
 
-You can also configure a format on any action class. `format` in an action class is analogous to `config.actions.format` in your app class, just as `config` in an action is analogous to `config.actions` in your app.
+You can also accept formats on any individual action class. `config.formats` in an action class is
+analogous to `config.actions.formats` in your app class.
```

Comment:
Just a random thought, but maybe "works the same way as" is more precise than "is analogous to", which suggests similarity with differences (at least to this non-native English speaker)